internal/fields/model.go

```diff
@@ -64,4 +75,40 @@ func (orig *FieldDefinition) Update(fd FieldDefinition) {
 		}
 		orig.Fields = updatedFields
 	}
+
+	if len(fd.MultiFields) > 0 {
+		updatedFields := make([]MultiFieldDefinition, len(orig.MultiFields))
+		copy(updatedFields, orig.MultiFields)
+		for _, newField := range fd.MultiFields {
+			found := false
+			for i, origField := range orig.MultiFields {
+				if origField.Name != newField.Name {
+					continue
+				}
+
+				found = true
+				updatedFields[i].Update(newField)
+				break
+			}
+			if !found {
+				updatedFields = append(updatedFields, newField)
+			}
+		}
+		orig.MultiFields = updatedFields
+	}
+}
+
+// MultiFieldDefinition describes a multi field with its properties.
+type MultiFieldDefinition struct {
+	Name string `yaml:"name"`
```

Comment:
Multi-fields don't have other properties, right?

Reply:
I'm wondering who can evaluate this. @ruflin Do you have any idea?

Reply:
Let's analyze what these models are used for:
* To import fields from ECS: Only `type` and `name` is defined there for multi fields. @ebeahan do you know if there are plans to add more properties to multi fields in ECS?
* To generate documentation: Should we document somehow these multi fields? Maybe as part of their parent fields? In any case we wouldn't need more properties beyond its name.
* To validate documented fields: Probably we don't need more properties, but maybe we need to add validation for multi fields if they start appearing in test documents.

Reply:
> * To validate documented fields: Probably we don't need more properties, but maybe we need to add validation for multi fields if they start appearing in test documents.

Fixed in 633f703e6e16a943203ff059df189ec955aee454.

Reply:
@mtojek What you mean by evaluate here? You mean how much we should support?

Reply:
Yeah, I mean which prorities are the high priority, which ones can be postponed for later, and which ones can be "forgotten".

Reply:
I would say the ones that exist already in integrations today we need to support, everything else we should do based on request.

Reply:
> * To generate documentation: Should we document somehow these multi fields? Maybe as part of their parent fields? In any case we wouldn't need more properties beyond its name.

Added multi-fields to documentation in https://github.com/elastic/elastic-package/pull/752/commits/fbb0809f1b91310c73bde8c658ea901d895d98d3, let me know what you think.

This will require to regenerate READMEs in packages using multi-fields.